### DISCUSSION TITLE
FIL-RetroPGF-1: Nominations for Protocol Research & Development

### DISCUSSION THREAD

**Discussion by kkarrancsu** [filecoin-project/community#693] [RetroPGF]
### Protocol Research & Development

This is the thread to nominate projects for the **_Protocol Research & Development_** category of FIL-RetroPGF-1. As a reminder, this category is intended to capture work to improve the Filecoin protocol, to include both theoretical contributions as well as implementations and prototypes are in scope.

### Nomination Template
When nominating a project, please answer the following questions in a comment to this discussion:

1. The project name
2. A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)
3. A link to the project’s GitHub or Twitter
4. Contact info for the project or project lead

### Nomination Considerations

Things to consider when nominating a project:
- Use your best effort to describe how this project has delivered impact to the Filecoin ecosystem in the previous six months (Oct 2023 - Mar 2024). Use the template provided in each category’s thread when nominating a project.
- Make sure to nominate projects in the relevant category. If you feel that a project spans multiple categories, please choose the one category that you feel it best aligns with. Do not nominate the same project across multiple categories! 
- Note that the organizers of FIL-RetroPGF-1 may choose to reassign a project’s category from its nominated one, if they feel it better aligns elsewhere. 
- Before nominating a project please check if this project has been nominated already, so we can avoid duplication.
RetroPGF is an experiment that Filecoin plans to iterate on regularly. Community feedback is key to helping the process improve with each round. We welcome your notes, reflections, and feedback at any step. Please use this discussion to submit feedback!

Project Nominations will close by Mar 31, 2024.

See [here](https://github.com/filecoin-project/community/discussions/689) for general guidelines on project nomination.

_Note: These threads will be moderated to only include project nominations. Off-topic comments that show support or hype for a project or meta-commentary on the program itself will be removed to keep the thread clean. If you want to show your excitement for a nomination, use the reaction feature!_

**Comment by jnthnvctr** [comment]
The project name: 
Banyan

A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)
- Banyan is one of the early teams seriously pushing enterprise adoption in the Filecoin ecosystem. Notable in their approach has been the open sourcing of their own research to the community: which has provided critical early feedback to core devs on gaps in the protocol, helped shape prioritization (to enable Filecoin to become a more critical component of an enterprise offering), and reduced the barrier of entry for new start ups looking to build enterprise offerings in the space). Example: https://www.youtube.com/watch?v=uU96bIyruwo
- Banyan has also been an active contributor to the creation of specs and standards to help enable composability between collab-apetitive teams: https://github.com/banyancomputer/on-chain-compliance-attestations

A link to the project’s GitHub or Twitter
https://github.com/banyancomputer

Contact info for the project or project lead
c@banyan.computer

**Comment by Joss-Hua** [comment]
**The project name:**
FIP-0021, FIP-0029

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**
- As is well known, "Secter quality" is the base of the Filecoin network consensus, and accurate values are a hallmark of the core contribution of SP in the network;
- To provide more flexibilities for Filecoin finance market, without this implementation, there would be no current lending market(DEFI);

**A link to the project’s GitHub or Twitter**
https://github.com/filecoin-project/FIPs?tab=readme-ov-file#fips
- https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0021.md
- https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md

**Contact info for the project or project lead**
venus@ipfsforce.com

**Comment by marco-storswift** [comment]
Project:
StorSwift

Description of impact in last 6 months:

1 StorSwift provides SP with an MPC-based wallet based on lotus repo, which improves the convenience of network users using the wallet, lowers the usage threshold, and improves the security of the wallet.
2 The wallet API based on lotus provides the offline sign function to ensure that the user's private key is not stored in the production environment and ensures the security of the user's large assets.
3 The team optimized the gas overhead of buildin Actor, enriched and improved the availability of the network.
A link to the Github / Twitter:
mpc ：https://github.com/storswiftlabs/imfil branch`online-v3-mpc`
offline-sign:https://github.com/storswift/lotus-offline-sign
network optimized:
https://github.com/filecoin-project/lotus/pull/8696
https://github.com/filecoin-project/lotus/pull/8854
https://github.com/filecoin-project/lotus/pull/8545
https://github.com/filecoin-project/go-state-types/pull/125
https://github.com/filecoin-project/builtin-actors/pull/722
https://github.com/filecoin-project/lotus/pull/11063
https://github.com/filecoin-project/lotus/pull/10928

Contact:
mfkrgxa@gmail.com marco@storswift.com

**Comment by jsoares** [comment]
**The project name**

[EC Finality Calculator](https://fastfinality.io/EC-Finality-Calculator.html)

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

Filecoin's Expected Consensus comes with probabilistic finality and a 900-epoch soft finality threshold. Up until the release of the EC Finality Calculator in Jan 2024, there was no rigorous way to determine the reversal probability of a given tipset/chain. Therefore, network participants like exchanges and L2 operators had no other option but to enforce a worst-case wait time of 100-900 epochs (i.e. _hours_) before clearing transactions or releasing funds. 

FRC-0089 introduces a new, entirely client-side method that allows participants to compute finality thresholds in view of the actual chain history. Under normal operating conditions, it brings down the confirmation from hours to ~15 minutes while maintaining the same level of security, greatly improving the UX for applications and for the overall network. The FRC specificies the calculator and is accompanied by a prototype stand-alone implementation that can be used _today_ by any interested party.

**A link to the project’s GitHub or Twitter**
https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0089.md
https://github.com/consensus-shipyard/ec-finality-calculator

**Contact info for the project or project lead**
jorge@protocol.ai (@jsoares), guy.goren@protocol.ai (@guy.goren)

**Reply by jsoares** [reply]
Some notes:
* This could fit under tooling too, but this seemed like the right place; please reassign if needed
* I'm self-nominating as one of the project members
* The project members are employed by PL but this has been a sort of "20% project" for us, outside of our core duties
* While the calculator is ready and usable today, PL isn't funding an out-of-the-box implementation in Filecoin clients, which would greatly increase usability, reach, and impact; we hope to use any rewards for the past work to fund a community bounty for said implementation

**Comment by ansermino** [comment]
### The project name

Forest

### A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)

- Highly performant Filecoin client running on Mainnet and testnets.
- The only non-Go Filecoin node in the ecosystem, written in Rust 🦀, directly interfacing with the FVM without FFI bindings.
- Supported all network upgrades in the past six months, with performant state migrations that helped bootstrap non-Forest nodes after the upgrade.
- A significant reduction in average resource consumption versus other implementations reduces the node operators' cost. Additionally, operations such as snapshot generation and full archival data exporting are to be done on minimal hardware
    - Able to produce snapshots with 4 core CPU, 10GiB memory; twice as fast as Lotus
    - 50% reduction in the size of archival data exports, with a slower rate of growth over time
- Contributions to network upgrades, including catching critical bugs and bringing sanity to the release process
- Direct cooperation and collaboration with Lotus and Venus maintainers.
- Enabled bootnode capabilities to bring more decentralization to the network

### A link to the project’s GitHub or Twitter

[github.com/ChainSafe/forest](http://github.com/ChainSafe/forest)

### Contact info for the project or project lead

[forest@chainsafe.io](mailto:forest@chainsafe.io)

<p align="center"><img width="129" alt="Forest" src="https://github.com/filecoin-project/community/assets/14164624/ef724512-a740-4d39-832a-5a9d0fef4dde"></p>

**Reply by jennijuju** [reply]
It was really great to see all the progress Forest has made in the past year, including starting to be one of the clients backing up the Fileocin calibration testnet, bootstrap nodes, and snapshot services! And working with forest devs on upgrade tests is super fun!

**Comment by geomatrick** [comment]
### The project name
Spark Protocol

### A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)
Spark samples file retrievals from Filecoin Storage Providers to improve the health and performance of the Filecoin Retrieval Market. 

Between October 2023 and March 2024, the number of (Station) nodes running Spark requests went from essentially 0 to ranging between 10k and 20k daily. Spark is now making around 20m retrieval requests a day to Filecoin Storage Providers and it is requesting roughly half a million different file CIDs each week.

In March, the Spark protocol added the ability to calculate retrieval success rate by Storage Provider, by linking up keys in IPNI and in on-chain records. This means that the Spark Protocol can now understand the retrieval performance of each Storage Provider at a file retrieval level.

N.B. Spark is making file CID retrievals not piece CID retrievals.
 
### A link to the project’s GitHub or Twitter
- filspark.com
- https://github.com/filecoin-station/spark


### Contact info for the project or project lead
spark@meridian.space

**Reply by momack2** [reply]
@patrickwoodhead is there a dashboard or leaderboard for SP success rate now?

**Reply by geomatrick** [reply]
There is a public dashboard but it has not yet been migrated into the Space Meridian domain yet. We will share as soon as it is ready. I will DM you the current link.

**Comment by nicola** [comment]
## Project name: **Future PoRep Research**

### A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)

Description
- The New PoRep Research WG has been researching a new PoRep and to ensure that it is secure
- In 2023/24 we rebooted research effort for designing a new proof of replication that could substitute SDR with the primary goal of achieving fast retrieval and secondary goal of achieving simpler proof system with lower hardware requirements.
- Outcomes from this work:
    - First, we de-risked the SDR to be secure in the cost model (after the Supranational improvement, the security margin was thought to be 2x - which would make SDR with current parameters only secure in the cost model for a few more years). We showed that there are tighter bounds which lead to a more concrete cost calculation, showing that the current security margin is instead 168x (this is a 84x improvement!). This means not only that SDR is secure for many years, but also that there is a large margin of improvement for new PoReps.
        - Better cost analysis via adding costs for storage [https://www.notion.so/pl-strflt/Better-Cost-Analysis-via-Adding-Costs-of-Storage-and-Computation-532e034e9389412099368ac0da22bf8c](https://www.notion.so/Better-Cost-Analysis-via-Adding-Costs-of-Storage-and-Computation-532e034e9389412099368ac0da22bf8c?pvs=21)
    - Second, we wrote the first summarized paper about SDR and designed SPR, a tighter construction with better bounds, which could be a stepping stone for future PoRep
        - Proofs of Space with Maximal Hardness https://eprint.iacr.org/2023/1530
    - Third, we produced several design documents for potential constructions
        - Why Better Predecessor Robustness or Depth Robustness are crucial: [https://www.notion.so/pl-strflt/Why-Better-Predecessor-Robustness-or-Depth-Robustness-are-Crucial-3237d78ed700402abb774d2628024cae?pvs=4](https://www.notion.so/3237d78ed700402abb774d2628024cae?pvs=21)
        - Unsealing vs Storing a second copy [https://www.notion.so/pl-strflt/Unsealing-vs-Storing-a-Second-Copy-5f9020e052164765887a5a77cf0fcad2](https://www.notion.so/Unsealing-vs-Storing-a-Second-Copy-5f9020e052164765887a5a77cf0fcad2?pvs=21)
- People: Leo, Carla, Anca, Matteo with help from Nicola, Irene, Luca
- Link to project:
    - Summary of the work done in 2023/24 [https://www.notion.so/pl-strflt/Retrospective-on-PoS-Research-2023-and-Status-0df524fc650a40ab8ec590636fa9f44d](https://www.notion.so/Retrospective-on-PoS-Research-2023-and-Status-0df524fc650a40ab8ec590636fa9f44d?pvs=21)

### Contact info for the project or project lead

This is on behalf of CryptoNet, whose new re-incarnation will be Public Good Crypto (website coming up soon)

nicola at filoz dot org

**Comment by irenegia** [comment]
I would like to nominate the “Practical PoRep Improvements WG” which did analysis to understand the pain points for Filecoin Storage Providers and acted on those shipping reports and FIPs that improved the status quo.

- **The project name: Practical PoRep Improvements WG**
- **Impact:**
    - **SnarkPack** (shipped): New sector onboarding method that reduces chain congestion;
    - **SnapDeal effort (shipped):**
        - A new method for updating a Committed Capacity (CC) sector with data (and deals) without resealing;
        - A blog post (together  with a detailed report and calculator) that helps SPs to understand the different costs of using SnapDeal;
    - **Synthetic PoRep (shipped)**: New PoRep with smaller storage overhead between PreCommit and ProveCommit;
    - **NI-PoRep (future work)**: New PoRep that removes on-chain interaction.
- **Links:**
    - https://github.com/filecoin-project/FIPs/blob/10c12f1585ba0c27a89ed16ab5693253b875587d/FIPS/fip-0013.md
    - https://github.com/filecoin-project/FIPs/blob/10c12f1585ba0c27a89ed16ab5693253b875587d/FIPS/fip-0019.md
    - https://filecoin.io/blog/posts/cc-sector-upgrade-guidelines-and-modeling/
    - https://github.com/filecoin-project/FIPs/blob/10c12f1585ba0c27a89ed16ab5693253b875587d/FIPS/fip-0059.md
    - https://github.com/filecoin-project/FIPs/blob/10c12f1585ba0c27a89ed16ab5693253b875587d/FIPS/fip-0090.md
- **Contact info**: [irene@filoz.org](mailto:irene@filoz.org)

**Comment by Kubuxu** [comment]
I would like to nominate the Verifiable Data Aggregation project, which resulted in FRC-0058 standardising Proof of DataSegment Inclusion (PoDSI).
- Project Name: **Verifiable Data Aggregation**
- Impact:
Verifiable Data Aggregation enables trust-less onboarding of small pieces of data, an area currently underserved in by Filecoin. Existing storage aggregators integrated it (see [Lighthouse](https://docs.lighthouse.storage/lighthouse-1/filecoin-virtual-machine/section-a)), and new applications are built on top of it. It is set to become a basis for L2 data markets, increasing the ease of use of Filecoin and opening new opportunities.
- Links:
  - [FRC-0058](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0058.md)
  - https://docs.lighthouse.storage/lighthouse-1/filecoin-virtual-machine/section-a
- Contact Info: kubuxu@filoz.org

**Comment by Schwartz10** [comment]
Project:
Lotus

Description of impact in last 6 months:
- The Lotus team has made countless contributions to the Filecoin core protocol and also maintains the Go Filecoin implementation (Lotus). For GLIF, the protocol research and development has significantly helped to make our defi application possible.
- Lotus has delivered ~15 network upgrades since mainnet liftoff and including implementing, testing and delivering large amount of the FIPs finalized, including but not limiting supporting FVM & FEVM, security fixes, snap deals & etc.

A link to the Github / Twitter:
https://twitter.com/lotus_web3

Contact:
[jennifer@filoz.org](mailto:jennifer@filoz.org)

**Comment by nicola** [comment]
Project:
Public Good Crypto (was CryptoNet)

Description of impact in the last 6 months:
* Cryptonet has been a very prolific research and protocol design team in the Filecoin ecosystem
* The protocol part of CryptoNet is going to FilOz, the research part is turning int Public Good Crypto
* This team plans to continue building a large community of researchers and do great research for the success of the Filecoin network (previously worked on SnarkPack, Testudo, Vector Commitments, Proof of Replication)

**Comment by gitdata001** [comment]
**The project name**
JiaoZiFS - A version control file system for data lineage & data collaboration.

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

Generative AI is driving increasing interest in storing and making sense of unstructured and semistructured data, along with multimodal data like images, video, and audio.

Soon, previously overlooked data sets like Zoom calls, performance review feedback, customer support tickets, and comments from online forums may provide value—with the corollary that enterprises will want to store them.

In parallel, important signals will emerge from wider domains of the enterprise, beyond traditional centers like product and engineering.

This explosion in the types of data enterprises store, together with the increased heterogeneity of data sources, is another key trend that’s reconfiguring the data infrastructure landscape.

In response to cost pressures and growing data volumes, especially in unstructured and heterogeneous data, enterprises are centralizing their data infrastructure into a data hub.

Long term, we think more and more data will be housed directly in object stores like JiaoZiFS, IPFS, and Filecoin. At the same time, we also foresee data infrastructure becoming more decentralized, with each team within an enterprise operating as a spoke off this data hub.

![image](https://github.com/filecoin-project/community/assets/33315004/815e3e8f-cad6-4c08-acfb-f75f958e3d8f)

JiaoZiFS currently supports IPFS as a back-end object storage, you only need to specify this parameter when creating a data repositiory in [https://cloud.jiaozifs.com](https://t.co/O6QMgyvRpX)
{"type":"ipfs","ipfs":{"url":"/dns/kubo-service.ipfs.svc.cluster.local/tcp/5001"}}

![image](https://github.com/filecoin-project/community/assets/33315004/62b32bc0-7130-4212-9a7b-72c47ea296db)

We deployed IPFS together with JiaoZiFS, you can view these files here:
https://ipfs.jiaozifs.com/ipfs/bafybeidf7cpkwsjkq6xs3r6fbbxghbugilx3jtezbza7gua3k5wjixpmba/#/filesThis also means:
IPFS is used for data storage and data synchronization, and JiaoZiFS is used as a  Git-Like data versioning semantic layer.

![image](https://github.com/filecoin-project/community/assets/33315004/0f418e91-be72-4388-a58b-4571131f7441)

This way you can use JiaoZiFS as the model registry to checkpoint the model in the training iteration. IPFS is used to synchronize model data sets between core data center and near/far Edge.

![image](https://github.com/filecoin-project/community/assets/33315004/4531c92e-c08c-4aec-a9df-fd8376fddccc)

Longer term, we believe each spoke will operate as a generative-AI-enabled agent, with the ability to pull data in real-time and answer queries autonomously, all while ensuring proper governance.



**A link to the project’s GitHub or Twitter**

https://github.com/GitDataAI/jiaozifs

**Contact info for the project or project lead**

info@gitdata.ai

**Comment by taoshengshi** [comment]
**The project name**
Mount - An Asset Creation,Issuance and Governance Protocol for Data DAOs

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**
https://docs.google.com/document/d/1ROCr5kEWvhweIrSOGqn393BTOD-5YZ1pmxVdfB0d9mY/edit#heading=h.htydihn1kc6i

**A link to the project’s GitHub or Twitter**
https://github.com/datadao-wg/mount-protocol

**Contact info for the project or project lead**
KEN Labs and taoshengshi01@gmail.com

**Comment by momack2** [comment]
**The project name**
The Filecoin Virtual Machine

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**
- FIPs 30, 31, 32, 48, 49, 50, 54, 55, 57 (which together implement user programmability via FVM and FEVM)
- The Filecoin Virtual Machine introduced user programmability into the Filecoin network - unlocking a massive community of builders and applications that make Filecoin much more rich, capable, and upgradable than before
- The FVM was the main building block which unlocked DeFI lease/loan markets (unblocking SP growth and access to collateral), L2 networks building on Filecoin, and on-chain storage solutions like Lighthouse, Eastore, and many others which unlock rich storage solutions

**A link to the project’s GitHub or Twitter**
- https://github.com/filecoin-project/ref-fvm
- https://twitter.com/fvmdev
- fvm.filecoin.io

**Contact info for the project or project lead**
@raulk @trruckerfling @momack2 @Stebalien

**Comment by jennijuju** [comment]
**The project name**

Filecoin Protocol Opportunity WG

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

This WG focuses on researching, proposing, reviewing, and auditing new FIPs, from early ideation/research to prototypes of new improvements to the Filecoin protocol, spec, etc. 

Collectively they have proposed >30 FIPs, to provide some features adopted or prototyped over the past 6 months:
- PoRep features (FIP0059, FIP0090)
- FVM enhancements (FIP0071, FIP0072, FIP0073, FIP0075)
- User programmability (FIP0076, FIP0083, FIP0062)
- Security and chain scalability (FIP0060, FIP0061)

**A link to the project’s GitHub or Twitter**


**Contact info for the project or project lead**

@jennijuju @nicola

**Reply by jennijuju** [reply]
Note: This WG is not yet "official." The goal is to acknowledge those who actively and consistently participate in Filecoin protocol development and motivate more developers to do the same.
A couple of us took a brief look at the proposed and finalized FIPs and FIP discussion and would like to thank @anorth, @ZenGround0, @nicola, @irenegia, @lucaniz, @Kubuxu, @Stebalien, @jennijuju, @arajasek, and @steven004 for contributing to this effort. We are likely missing some folks—please feel free to nominate those developers who have constantly contributed and impacted this front!

**Reply by taoshengshi** [reply]
@hunjixi - former core developer of go-filecoin (venus)

**Comment by vkalghatgi** [comment]
Project Name: CryptoEcon Core-Dev Contributions 

Description of Impact

CryptoEconLab members are core devs and specialists in Filecoin cryptoeconomic design and improvement. Each report (linked below) addressed critical research and development needs in the Filecoin economy within the last six months, helped the Filecoin ecosystem better understand the current state of the Economy, and whether any actions should be taken to modify/update Filecoin’s CryptoEconomics. 

Selected Works
[DDO](https://hackmd.io/@cryptoecon/r1vWXdrL6?type=view)
[Universal Multipliers](https://hackmd.io/gzftJx5eQDKhv4-UEubSgg?view=)
[Filecoin 2024 SitRep](https://hackmd.io/@cryptoecon/rkgJ9_NYT?type=view)
[FIP0081](https://github.com/filecoin-project/FIPs/blob/284ca3b674576c62558796d2328a23f7a3157c7e/FIPS/fip-0081.md)
[Associated Analysis](https://docs.google.com/document/d/1uV33W0CUjOgHbUUhCWB7-LZkL10fVNRYr1xSjEbU6ls/edit#heading=h.3qrism6s35w5)


[Github](https://github.com/celtd)
[Twitter](https://twitter.com/cryptoeconlab)

Contact: [advisory@cel.build](mailto:t@cel.build)

**Comment by aschmahmann** [comment]
**Project name**

FRC-0069

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

Identified usability issues with how piece CIDs are used and reached consensus on an easy to upgrade to alternative. This format is currently in use by web3.storage and Spade within their storage pipelines and has set the stage for upgrades in Boost as well as other Filecoin applications that work with Filecoin pieces and piece CIDs.

**A link to the project’s GitHub or Twitter**

https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0069.md

**Contact info for the project or project lead**
adin@ipshipyard.com

**Comment by aschmahmann** [comment]
**The project name**
filexp

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

FilExp is Filecoin's first read-enabled light client, and in the past 6 months demonstrated the ability to respond to verifiably answer EthRpc calls by dynamically pulling data from the Filecoin (either the Filecoin network itself or from Filecoin chain data that has been stored with Filecoin SPs).

While it has some users today, the primary impact has been in demonstrating that the Filecoin protocol has many of the pieces in place to make light clients usable **today** while also identifying the areas of code, documentation, and infrastructure that would be needed to make Filecoin light clients not only usable in many situations. It has also been a demonstration of how Filecoin can itself enable light clients for other networks by storing and serving light client data with no need for protocol-specific smarts on the Filecoin SPs.

**A link to the project’s GitHub or Twitter**
https://github.com/aschmahmann/filexp

**Contact info for the project or project lead**
adin.schmahmann@gmail.com

**Reply by jennijuju** [reply]
+1 - First, having a light client prototype, well, a working one is cool! And afaik many folks have been using this tool to manage their msigs!

**Comment by dannyob** [comment]
# UCAN WG

## Project name: UCAN Working Group

User Controlled Authorization Networks (UCANs) are a way of doing authorization (“what you can do”) where users are fully in control, and represent a decentralization-centric version of controlling and delegating access. UCAN is an ecosystem-wide standard that has been adopted by a number of different projects, notably Fission’s set of IPFS-related products, and NFT Storage and Web3 Storage’s new API and backend.

## Positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024):

The UCAN standard was developed over a period of years, but the pre-1.0 version of the spec (v0.10) was finalized in September 2023, allowing the work to be built on by all its users. 

Web3 Storage’s move to a new w3up API, [built on UCAN](https://blog.web3.storage/posts/intro-to-ucan)], took place on  20 November 2023. The benefit of working from a ecosystem-agreed standard meant that not only could this switch be made with some confidence, but also that other organizations could work compatibly with Web3/NFT storage using the same standard: a vital step given the amount of interoperability that an authorization system needs for the widest adoption and use.

From October to the present, the working group has concentrated on getting the standard to 1.0. 

Convening, documenting, resolving and maintaining standards is a classic public good: everyone benefits, but the rewards for individuals to contribute are far smaller than the total advantage for all.

I nominate the UCAN WG, in the hope that we can send a strong signal that standard working groups are key coordinating feature in the ecosystem, and its connections to other parts of the digital space.

## Links

- Github: https://github.com/ucan-wg
- https://ucan.xyz/
- Current work: https://github.com/orgs/ucan-wg/discussions

## Contact info for the project or project lead:

Brooklyn Zelenka, Fission,  brooklyn@fission.codes

Daniel Holmgren, Bluesky,   dtholmgren@gm-ail.com

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This week marks an important milestone for the FIL-RetroPGF-1 effort.  Nominations for projects closed on Mar 31, 2024.  We received over 150 nominations for projects across the various categories that were eligible for FIL-RetroPGF-1 funding. 

Thank you to the community for taking the time and effort to put these nominations together. The projects nominated reflect all of the incredible work and progress that the Filecoin community have made together.

As FIL-RetroPGF-1 organizers, our next step is to review the nominations and select eligible projects to apply formally to be considered for FIL-RetroPGF-1 funding. We will be using the criteria outlined [in this document](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine whether a nominated project should be asked to apply.  Nominated projects will be informed soon, and applications are currently slated to open on Apr 8th.

Thank you, stay tuned, and check back to this page and main our [RetroPGF landing page](https://fil-retropgf.notion.site/FIL-RetroPGF-4b6f5358440043c8bb1bf53f0297541e) for future developments!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

The FIL-RetroPGF organizers have spent the last week reviewing the nominations submitted in each of the eligible categories. We used the criteria outlined [here](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine which projects would be invited to apply in this round. Our assessment of eligibility is based on our best interpretation of the information provided in the nomination but tended to bias towards inclusivity, deferring judgement to badgeholders as much as possible. 

The following [page](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) contains a list of projects that are being invited to apply to be eligible for FIL-RetroPGF-1 funding.

Applications are open from 8 Apr 2024 to 22 Apr 2024. We encourage you to submit your application [here](https://www.fil-retropgf.io/) as soon as possible!

Best regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This is a gentle reminder that the application submission deadline is fast approaching. If you are a project invited to apply and have not already done so, please submit your applications [here](https://www.fil-retropgf.io/) by Apr 22.

We have updated the [Round 1 - Invited Applications](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) page to include a POC. Additionally, we have published [the following guidelines](https://fil-retropgf.notion.site/Project-Application-Guidelines-4cf9df76d0f74da9b02cd33f09cbd5b4) on how to approach the application process and included an FAQ at the bottom.

Best Regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

We have extended the deadline to submit your applications. If you haven’t already done so, please submit your application [here](http://fil-retropgf.io/) by 12PM UTC Apr 24, 2024. There will be no further extensions.

Best regards.

**Comment by kkarrancsu** [comment]
Dear Filecoin community,
Thank you for your continued patience. The organizers have finished tallying the results and computing the allocations for each project, according to the scoring rules that were previously set forth. Please visit the following resources for the results:

- [FIL-RetroPGF-1 Results overview](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628)
- [Results Spreadsheet](https://docs.google.com/spreadsheets/d/15tQZU4yfzCCso_dd5NGXfSy6XB2pmrEZzZrG_TEIKyY/edit#gid=1610641018)
- [Deep dive into the results.](https://medium.com/p/7e5a0bcdba08)

We will contact project POCs in the coming days to coordinate distribution. A huge thank you to the entire Filecoin ecosystem for coming together to make this first round successful. Be on the lookout for Round 2!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

Thank you for your continued patience. We are now in the next phase of FIL-RetroPGF-1—funds distribution. The [Open Impact Foundation](https://openimpact.foundation/) will coordinate distribution and contact projects [that have been allocated funds](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628?pvs=74) from the FIL-RetroPGF-1 process.

Please be on the lookout in your inbox for emails coming from hello@openimpact.foundation

Best Regards,
